warzone/PlayerDriver.cpp

```diff
@@ -0,0 +1,30 @@
+#include "PlayerDriver.h"
+#include "Orders.h"
+
+
+int PlayerDriver::main() {
+	Player* p1 = new Player();
+
+	cout << "Player Territories are: ";  //returns all the player's territories 
+	p1->getTerritories();
+
+	cout << "Player Hand is: "<<endl;  //returns the cards in the player's hand
+	p1->getHand();
```

Comment:
Do we want to actually print this stuff (territories, hand)? Right now these aren't getting printed

Reply:
I thought it might be nice to have them though I know it wasn't a requirement. I'll delete that part though